src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1225,7 +1599,7 @@ by rw [CNF_rec, dif_neg o0]
   in the base-`b` expansion of `o`.
 
     CNF b (b ^ u₁ * v₁ + b ^ u₂ * v₂) = [(u₁, v₁), (u₂, v₂)] -/
-noncomputable def CNF (b := omega) (o : ordinal) : list (ordinal × ordinal) :=
+def CNF (b := omega) (o : ordinal) : list (ordinal × ordinal) :=
```

Comment:
What happened here?

Reply:
At some point, I removed a bunch of redundant `noncomputable`s from my definitions, and that one seems to have slipped by. It already got merged into `master` in PR #10979, though. Doesn't seem to have any negative consequences.